cmd/root.go

```diff
@@ -143,62 +187,57 @@ any client SSL certificates.`,
 
 	cmd.RunE = func(*cobra.Command, []string) error { return runSignalWrapper(c) }
 
+	// Override Cobra's default messages.
+	cmd.PersistentFlags().BoolP("help", "h", false, "Display help information for cloudsql-proxy")
+	cmd.PersistentFlags().BoolP("version", "v", false, "Print the cloudsql-proxy version")
+
 	// Global-only flags
 	cmd.PersistentFlags().StringVarP(&c.conf.Token, "token", "t", "",
-		"Bearer token used for authorization.")
+		"Use bearer token for authentication.")
```

Comment:
nit: Let's sidestep authZ vs authN here for now (same for the others) 
```suggestion
		"Use bearer token as a source of IAM credentials.")
```

Reply:
Done.